### PR TITLE
Added missing definition for, fromEuler(), to Quaternion.

### PR DESCRIPTION
**Pull request by surtr-isaz**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: http://glmatrix.net/docs/module-quat.html
- [x] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by RyanCavanaugh** [comment]
This PR has been open and unchanged 5 days without signoff or complaint. This will be merged by a maintainer soon if there are no objections.

**Review by nbabanov — APPROVED** [review]

File: types/gl-matrix/index.d.ts
Comment (DanielRosenwasser) [inline types/gl-matrix/index.d.ts]: Could you enable linting to in `tslint.json` to correctly parse the header? It should only be versioned at 2.4 (major.minor, not patch).

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]

**Comment by RyanCavanaugh** [comment]
@surtr-isaz Please address comments from the code reviewers.

File: types/gl-matrix/index.d.ts
Comment (surtr-isaz) [reply types/gl-matrix/index.d.ts]: Fixed. Thanks for pointing that out!

**Comment by surtr-isaz** [comment]
Updated as per @DanielRosenwasser's comment.

**Comment by RyanCavanaugh** [comment]
@DanielRosenwasser - Thanks for your review of this PR! Can you please look at the new code and update your review status if appropriate?

**Comment by DanielRosenwasser** [comment]
Thanks for your patience!

**Review by DanielRosenwasser — APPROVED** [review]

**Comment by surtr-isaz** [comment]
@DanielRosenwasser No worries, thanks!

**Pull request merged**
